modules/test/playwright/tests/portal-workflow-task-web/main/metrics.spec.ts

```diff
@@ -0,0 +1,201 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../../fixtures/apiHelpersTest';
+import {dataApiHelpersTest} from '../../../fixtures/dataApiHelpersTest';
+import {loginTest} from '../../../fixtures/loginTest';
+import {workflowPagesTest} from '../../../fixtures/workflowPagesTest';
+import clearInvalidUserNotifications from '../../../utils/clearInvalidUserNotifications';
+import getBasicWebContentStructureId from '../../../utils/structured-content/getBasicWebContentStructureId';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	dataApiHelpersTest,
+	loginTest(),
+	workflowPagesTest
+);
+
+interface CreatedEntities {
+	blogPosts?: TBlogPost[];
+}
+
+const createdEntities: CreatedEntities = {};
+
+test.beforeEach(async ({apiHelpers, configurationTabPage}) => {
+	await configurationTabPage.goTo();
+
+	await configurationTabPage.assignWorkflowToAssetType(
+		'Single Approver',
+		'Blogs Entry'
+	);
+
+	const site = await apiHelpers.headlessSite.getSiteByERC('L_GUEST');
+	const blogPosts: TBlogPost[] = [];
+	for (let i = 1; i <= 21; i++) {
+		blogPosts.push(
+			await apiHelpers.headlessDelivery.postBlog(site.id, {
+				headline: `Blogs Entry ${i}`,
+			})
+		);
+	}
+
+	createdEntities.blogPosts = blogPosts;
+});
+
+test.afterEach(async ({apiHelpers, configurationTabPage, page}) => {
+	await configurationTabPage.goTo();
+
+	await configurationTabPage.unassignWorkflowFromAssetType('Blogs Entry');
+
+	if (createdEntities.blogPosts?.length) {
```

Comment:
if using the `apiHelpers.data.push` pattern works, then this would not be necessary.